packages/xo-server-backup-reports/src/index.js

```diff
@@ -679,6 +679,11 @@ class BackupReportsXoPlugin {
           status: nagiosStatus,
           message: nagiosMarkdown,
         }),
+      xo.sendIcinga2Status !== undefined &&
+        xo.sendIcinga2Status({
+          status: nagiosStatus, // Same values for Icinga2 status
```

Comment:
What if one day it is not?

Reply:
I wanted to use my exported values:
```js
export const OK = 0
export const WARNING = 1
export const CRITICAL = 2
export const UNKNOWN = 3
```

But I'm not sur how to do it. Since the plugin can be not loaded.